docs/fundamentals/read-operations.txt

```diff
@@ -175,6 +176,124 @@ Use the following syntax to run a find operation that matches all documents:
    more information about ``take()``, see the :ref:`laravel-modify-find` section of this
    guide.
 
+.. _laravel-retrieve-text-search:
+
+Search Text Fields
+------------------
+
+You can perform a text search by using the :manual:`$text
+</reference/operator/query/text>` operator followed
+by the ``$search`` field in your query filter that you pass to the
+``where()`` method. The ``$text`` operator performs a text search on the
+text-indexed fields. The ``$search`` field specifies the text to search for.
+
+A text search retrieves documents that contain a **term** or a **phrase** in the
+text-indexed fields. A term is a sequence of characters that excludes
+whitespace characters. A phrase is a sequence of terms with any number
+of whitespace characters.
+
+.. note::
+
+   Before you can perform a text search, you must create a :manual:`text
+   index </core/indexes/index-types/index-text/>` on
+   the text-valued field. To learn more about creating
+   indexes, see the :ref:`laravel-eloquent-indexes` section of the
+   Schema Builder guide.
+
+After building your query with the ``where()`` method, chain the ``get()``
+method to retrieve the query results.
+
+This example calls the ``where()`` method on the ``Movie`` Eloquent model to
+retrieve documents in which the ``plot`` field contains the phrase
+``"love story"``:
+
+.. tabs::
+
+   .. tab:: Query Syntax
+      :tabid: query-syntax
+
+      Use the following syntax to specify the query:
+
+      .. literalinclude:: /includes/fundamentals/read-operations/ReadOperationsTest.php
+         :language: php
+         :dedent:
+         :start-after: start-text
+         :end-before: end-text
+
+   .. tab:: Controller Method
+      :tabid: controller
+
+      To see the query results in the ``browse_movies`` view, edit the ``show()`` function
+      in the ``MovieController.php`` file to resemble the following code:
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+            :language: php
+
+            class MovieController
+            {
```

Comment:
Suggestion:
If it makes it easier to test, I think you can omit the class and function declarations since it's already described as being inside the show() function of the MovieController.php file.
I would not recommend spending much effort on this though since it shouldn't impact the user's understanding.

Reply:
Leaving as is, since the other examples on this page use the same format for the code examples.